cfme/tests/generic_objects/test_instances.py

```diff
@@ -3,17 +3,78 @@
 import fauxfactory
 import pytest
 
+
 from cfme import test_requirements
+from cfme.base.login import BaseLoggedInPage
 from cfme.services.myservice import MyService
-from cfme.utils.appliance import ViaREST
+from cfme.rest.gen_data import categories as _categories, tags as _tags
+from cfme.utils.appliance import ViaREST, ViaUI
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.rest import assert_response
 from cfme.utils.update import update
-from fixtures.pytest_store import store
 
-pytestmark = [test_requirements.generic_objects]
+
+pytestmark = [
+    test_requirements.generic_objects,
+    pytest.mark.uncollectif(lambda appliance: appliance.version < "5.9",
+                            reason="5.8 appliance doesn't support generic objects")
+]
+
+
+@pytest.yield_fixture()
+def definition(appliance):
+    with appliance.context.use(ViaREST):
+        definition = appliance.collections.generic_object_definitions.create(
+            name='rest_generic_class{}'.format(fauxfactory.gen_alphanumeric()),
+            description='Generic Object Definition',
+            attributes={'addr01': 'string'},
+            associations={'services': 'Service'},
+            methods=['add_vm', 'remove_vm']
+        )
+        assert definition.exists
+        yield definition
+        definition.delete()
+
+
+@pytest.fixture()
```

Comment:
**REQUIRED**
Please use the same fixture types across the module. You started from `@pytest.yield_fixture`